### DISCUSSION TITLE
文明6 自动退出

### DISCUSSION THREAD

**Discussion by github342655596** [Gamuxorg/bbs#209] [游戏或网站运行反馈]
开游戏进去，载入画面，然后自动退出。
owl@linux:~/下载$ ./Civilization\ VI.appimage 
/tmp/.mount_CiviliF4bW0k/usr/bin//start.sh：行 6:  3342 已杀死               ./Civ6Sub
owl@linux:~/下载$

**Comment by hyh19962008** [comment]
解压运行

Best

________________________________

**Comment by wangling12** [comment]
是否看过置顶issue?另外,你用的是什么发行版?

**Comment by owl342655596** [comment]
还是不行哦，我都放弃了。

**Comment by hyh19962008** [comment]
试试freeciv吧

Best

________________________________

**Comment by wangling12** [comment]
ubuntu20.04是吧,回头我会测试文明5和6的.

**Comment by wangling12** [comment]
文明5在ubuntu20.04下确实有点问题,18.04下正常,抽空看看20.04对32位库做了啥限制吧.

**Comment by owl342655596** [comment]
谢啦

**Comment by hyh19962008** [comment]
文明5试试 解压后，删除libm.so.6，看能否运行

**Comment by owl342655596** [comment]
文明5可以运行了，只是没中文。
谢谢

**Comment by luwenhegithub** [comment]
@owl342655596 怎么解决的啊？

**Comment by owl342655596** [comment]
> @owl342655596 怎么解决的啊？

解压后删除libm.so.6

**Comment by luwenhegithub** [comment]
> > @owl342655596 怎么解决的啊？
> 
> 解压后删除libm.so.6

唉，可惜在Manjaro上不好使

**Comment by wangling12** [comment]
> 唉，可惜在Manjaro上不好使

可以帖报错看看

**Comment by luwenhegithub** [comment]
> > 唉，可惜在Manjaro上不好使
> 
> 可以帖报错看看

之前的删了，后来又重新下了一个试试
一开始的报错是
```shell
[ Applications]$ ./Civilisation5_i386.appimage 
/tmp/.mount_CiviliBcHp1T/unionfs: error while loading shared libraries: libfuse.so.2: cannot open shared object file: No such file or directory
/tmp/.mount_CiviliBcHp1T/AppRun:行23: ./Civ5XP: 没有那个文件或目录
```
然后下了个fuse-libs-2.9.7-12.el8.i686

然后用

```shell
[ Applications]$ LD_LIBRARY_PATH=/home/xxxx/下载/fuse-libs-2.9.7-12.el8.i686/usr/lib/ ./Civilisation5_i386.appimage 
```
就闪退了，发现程序已经解压，进入解压路径执行
```shell
[luwenhe@lu-pc Civilisation5]$ ./Civilisation5_gamux_mountdir/Civ5XP
./Civilisation5_gamux_mountdir/Civ5XP: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
```
然后用
```shell
[Civilisation5]$ LD_LIBRARY_PATH=./Civilisation5_gamux_mountdir ./Civilisation5_gamux_mountdir/Civ5XP
```
结果就是没有任何反映就退出了

删掉libm.so.6也是同样的表现

**Comment by hyh19962008** [comment]
libfuse 试试用64位的

Best

________________________________
